module3/src/main/java/com/epam/esm/module3/controller/hateoas/impl/UserHateoas.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.module3.controller.hateoas.impl;
+
+import com.epam.esm.module3.controller.UserController;
+import com.epam.esm.module3.controller.dto.UserDto;
+import com.epam.esm.module3.controller.hateoas.Hateoas;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Component
+public class UserHateoas implements Hateoas<UserDto> {
+    private final Class<UserController> CONTROLLER = UserController.class;
```

Comment:
Either make this field private **static** final, or rename it to lower case. Do it for all Hateos implementations